docker-compose.yml

```diff
@@ -14,30 +14,28 @@ services:
 
   web:
     extends: base
-    command: web -b 0.0.0.0:8000
+    command: web -b 0.0.0.0:8000 --reload
```

Comment:
In general `--reload` is really more for a development server so this should probably be removed.

Reply:
You're right, this will be removed.